### ISSUE TITLE
Enabling Win_ShowConsole in Game

### ISSUE THREAD

**Issue by lul3kk** [etlegacy/etlegacy#1584]
Hello developers.

In ETLegacy, the "Viewlog" console is disabled by default.
I would like to ask you to add the option to enable it in the in-game settings. 
Some players, use this console in the old ET for communication. You know - minimized ET, only the console launched, plus a web browser. Thanks to the console, I can copy links, e.g. from YouTube, and also send them to the server.
There are players that rely on this console more than in-game chat, and are not in depth with setting cvars.

Greetings
Lul3k
![Viewlog console](https://user-images.githubusercontent.com/52976621/108004627-a8d37380-6ff6-11eb-9ba4-5e17a9106a37.png)

**Comment by ensiform** [comment]
You can paste in the in game console as well.

**Comment by lul3kk** [comment]
> You can paste in the in game console as well.

But i can't have option copying from game

**Comment by rafal1137** [comment]
![obraz](https://user-images.githubusercontent.com/2415096/108118691-03c0a580-709f-11eb-9a06-f0222243717d.png)

Console is shown but there is no input and output from game

1447-gcb884aa has working console so we start from here.

**Comment by jackeri** [comment]
Not sure what the issue is here.. 
![image](https://user-images.githubusercontent.com/2070060/108131661-49d73280-70ba-11eb-8ff3-774f0bd46f52.png)

The console works just fine, if you really need to copy something from the console, its easier just to condump the console.

**Comment by rafal1137** [comment]
@jackeri I just want to inform you that this commit cb6e9d83cd44cc61ada3a2d602e1ec4be2a49fce is responsible for viewlog not working for me. Did not digged in to what could exactly went wrong but its a start point.

**Comment by rafal1137** [comment]
So after digging throu changes this one is breaking it for me
https://github.com/etlegacy/etlegacy/commit/cb6e9d83cd44cc61ada3a2d602e1ec4be2a49fce#diff-b3791e0a1b3d230f0d2381b36fa6fae08d582a9893e7a29a1d04b944c1ea8eecL283

**Comment by rafal1137** [comment]
> 
> 
> ![obraz](https://user-images.githubusercontent.com/2415096/108118691-03c0a580-709f-11eb-9a06-f0222243717d.png)
> 
> Console is shown but there is no input and output from game

This one can be ignored. I was looking why console does not work and I did get into commented code in 
Sys_CreateConsole

> /*
	ShowWindow(s_wcd.hWnd, SW_SHOWDEFAULT);
	UpdateWindow(s_wcd.hWnd);
	SetForegroundWindow(s_wcd.hWnd);
	SetFocus(s_wcd.hwndInputLine);
	s_wcd.visLevel = 1;
	*/

I did uncomented it and compiled which turned as image above with no input and output.

**Comment by jackeri** [comment]
The problem with this request is that the viewlog cvar is a cheatprotected cvar, not because its an inherently dangerous but because what it causes is a creation of another window and a lot extra string processing. So in the hands of people that do not know how to just set the viewlog to visible from startup vars do not understand either that having a game fullscreen and then opening a different window will cause the fullscreen one to minimize and lose focus and possibly hinder framerate.

Basically, if you need the viewlog, then you should be able to open it from the startup vars. If you are not able to do so, then clearly you don't need it either.

That said, I'd happy to accept pull requests or a ticket for the ability to copy text from the ingame console.
